scripts/ui/inventory/item.gd

```diff
@@ -0,0 +1,41 @@
+extends Node
+
+var item_name
+var item_quantity
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
+	var rand_val = randi() % 3
+	if rand_val == 0:
+		item_name = "Repair Kit"
+	elif rand_val == 1:
+		item_name = "Med Kit"
+	else:
+		item_name = "Bandage"
+	$TextureRect.texture = load("res://assets/ui/inventory_ui/item_icons/" + item_name + ".png")
+	var stack_size = int(JsonData.item_data[item_name]["StackSize"])
+	item_quantity = randi() % stack_size + 1
+	if stack_size == 1:
+		$Label.visible = false
+	else:
+		$Label.text = str(item_quantity)
+
+func set_item(nm, qt):
+	item_name = nm
+	item_quantity = qt
+	$TextureRect.texture = load("res://assets/ui/inventory_ui/item_icons/" + item_name + ".png")
```

Comment:
this is a bbaaaaaaaaaaaaaaaaaad idea and another reason why we should use resources. Would let us embed images in the resources without having to do this. Also means we wouldn't need to go through and edit a billion load lines if we move a file.